.gitmodules

```diff
@@ -15,24 +15,24 @@
 [submodule "kubelet"]
 	path = kubelet
 	url = https://github.com/openshift/kubernetes
-	branch = release-4.15
+	branch = release-4.16
 [submodule "kube-proxy"]
 	path = kube-proxy
 	url = https://github.com/openshift/kubernetes
-	branch = sdn-4.15-kubernetes-1.28.3
+	branch = sdn-4.16-kubernetes-1.28.3
```

Comment:
sdn-4.16-kubernetes-1.28.3 is not available yet, please create a ticket to move to it when it's created.